executor/src/witgen/query_processor.rs

```diff
@@ -241,6 +249,7 @@ impl<'a, 'b, 'c, T: FieldElement, QueryCallback: super::QueryCallback<T>> Symbol
         value: Arc<Value<'a, T>>,
     ) -> Result<(), EvalError> {
         // TODO allow "next: true" in the future.
+        // TODO allow assigning to publics in the future
```

Comment:
Although for those, the row is non-sensical

Reply:
Yes. But we should support it somehow, because it would give us public inputs!